google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ReadModifyWriteRow.java

```diff
@@ -50,10 +58,33 @@ public static ReadModifyWriteRow create(@Nonnull String tableId, @Nonnull String
     return new ReadModifyWriteRow(tableId, ByteString.copyFromUtf8(key));
   }
 
+  /**
+   * Creates a new instance of the ReadModifyWriteRow on an authorized view.
+   *
+   * <p>See {@link com.google.cloud.bigtable.admin.v2.models.AuthorizedView} for more details about
+   * an AuthorizedView.
+   */
+  public static ReadModifyWriteRow createForAuthorizedView(
+      @Nonnull String tableId, @Nonnull String authorizedViewId, @Nonnull String key) {
```

Comment:
Preconditions.checkNotNull(authorizedViewId, "authorized view id can't be null");